<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PocketRasterForm</class>
 <widget class="QWidget" name="PocketRasterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="1">
    <widget class="QLabel" name="labelPass">
     <property name="enabled">
      <bool>true</bool>
     </property>
<property name="text">
      <string>Profile Pass:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="DoubleSpinBox" name="dsbxAcc">
     <property name="suffix">
      <string> mm/c²</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="3">
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Aligin</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbOutside">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Outside</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbInside">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Inside</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rbNormal">
          <property name="text">
           <string>Milling</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbFast">
          <property name="text">
           <string>HLDI</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Direction</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbClimb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Climb</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbConventional">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Conventional</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" rowspan="3">
      <widget class="QLabel" name="lblPixmap">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/toolpath/offset_climb.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="ToolSelectorForm" name="toolHolder1" native="true"/>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="cbxPass">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>No Profile Pass</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>First</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="labelAcc">
<property name="text">
      <string>Acceleration:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelAngle">
     <property name="enabled">
      <bool>true</bool>
     </property>
<property name="text">
      <string>Raster Angle:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="DoubleSpinBox" name="dsbxAngle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> °</string>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ToolSelectorForm</class>
   <extends>QWidget</extends>
   <header location="global">tool_selectorform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
